### PR TITLE
let monospace text style be monospaced

### PR DESCRIPTION
**Pull request by winne42**
it's a bit weird if the monospace text style is not really monospaced and not used in the font style example

### CODE REVIEW

File: examples/custom_font_style/src/main.rs [hunk 1]
```diff
@@ -36,7 +36,7 @@ fn content(ui: &mut egui::Ui) {
     ui.label(LOREM_IPSUM);
     ui.add_space(15.);
     ui.label(RichText::new("Sub Heading").text_style(heading2()).strong());
-    ui.label(LOREM_IPSUM);
+    ui.label(RichText::new(LOREM_IPSUM).text_style(TextStyle::Monospace));
```
Comment (ItsEthra) [inline examples/custom_font_style/src/main.rs:39]: Maybe `ui.monospace(LOREM_IPSUM)` would be a bit cleaner?

**Review by emilk — APPROVED** [review]

File: examples/custom_font_style/src/main.rs (hunk 1 above)
Comment (winne42) [reply examples/custom_font_style/src/main.rs]: 👍  changed

**Comment by winne42** [comment]
@ItsEthra @emilk I don't know about your contribution guidelines, does @ItsEthra have to resolve the conversation? Or could this just be merged? Thx!

**Comment by emilk** [comment]
@winne42 no, I just have been busy with other stuff - sorry!

**Comment by winne42** [comment]
👍🙏

**Pull request merged**